media-gfx/prusaslicer/prusaslicer-2.3.0.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit cmake desktop wxwidgets xdg-utils
+
+MY_PN="PrusaSlicer"
+
+DESCRIPTION="A mesh slicer to generate G-code for fused-filament-fabrication (3D printers)"
+HOMEPAGE="https://www.prusa3d.com/prusaslicer/"
+SRC_URI="https://github.com/prusa3d/${MY_PN}/archive/version_${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="AGPL-3 Boost-1.0 GPL-2 LGPL-3 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="gui test"
+RESTRICT="!test? ( test )"
+
+# tests fail to link with USE=-gui, bug #760096
+REQUIRED_USE="test? ( gui )"
+
+RDEPEND="
+	dev-cpp/eigen:3
+	dev-cpp/tbb
+	>=dev-libs/boost-1.73.0:=[nls,threads]
+	dev-libs/cereal
+	dev-libs/expat
+	dev-libs/gmp:=
+	dev-libs/miniz
+	dev-libs/mpfr:=
+	>=media-gfx/openvdb-5.0.0
+	media-libs/ilmbase:=
+	media-libs/libpng:0=
+	media-libs/qhull:=[static-libs]
```

Comment:
and there's no way to avoid the static libs?

Reply:
qhull upstream states that libqhullcpp.a is intended to be static-only, and prusaslicer needs both libqhull_r.so and libqhullcpp.a

Before it was using the bundled copy, and I don't think that's better.

Reply:
Doesn't get rid of it but moved [static-libs] to DEPEND only.